File: Java/UberX.java
class UberX extends Car {
    String brand;
    String model;
    

    public UberX(String license, Account driver , String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    @Override
    void A() {
        super.A();
        System.out.println("Model: " + model + ",  brand: " + brand);
    }

}

File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }
    public function PrintDataCar(){
        echo"
        License: $this->license
        driver: {$this->driver->name} 
        Modelo: $this->model
        Marca: $this->brand 
        ";
      }


}
?>